inc/namespace.php

```diff
@@ -0,0 +1,314 @@
+<?php
+
+namespace Altis\Security\Browser;
+
+use Altis;
+use WP_Dependencies;
+use WP_Error;
+
+const INTEGRITY_DATA_KEY = 'altis_integrity_hash';
+const INTEGRITY_HASH_ALGO = 'sha384';
+const INTEGRITY_CACHE_GROUP = 'altis_integrity';
+
+/**
+ * Bootstrap.
+ *
+ * @param array $config {
+ *   @var bool $automatic-integrity Enable automatic generation of integrity hashes?
```

Comment:
This could be more explicit, eg. something along the lines "Enables automatic generation of integrity hashes if set to true".